ausroller/config.py

```diff
@@ -0,0 +1,119 @@
+#encoding: utf-8
+import argparse
+import sys
+import os
+import json
+import logging
+import ConfigParser
+
+
+class Configurator(object):
```

Comment:
The name sounds wrong. This is a configuration or a context but it does not configure anything, but holds configuration values.